pkg/helpers/command.go

```diff
@@ -4,14 +4,19 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"os"
 	"os/exec"
 
 	"github.com/sirupsen/logrus"
 )
 
+type RunCommandWithWriterOptions struct {
+	Env map[string]string
+}
+
 // RunCommandWithWriter runs a the provided command. The stdout of the command is
 // written to the provider writer.
-func RunCommandWithWriter(to io.Writer, bin string, args ...string) error {
+func RunCommandWithWriter(opts RunCommandWithWriterOptions, to io.Writer, bin string, args ...string) error {
```

Comment:
why not just pass it as another param and support it for both `RunCommandWithWriter` and `RunCommand`?

Reply:
ultimately reverted the above and made a more generic `RunCommandWithOptions` in https://github.com/replicatedhq/embedded-cluster/pull/574/commits/ed3b18e817688dbe7236d55f11a0dd4cb7aa8143